package/MDAnalysis/core/groups.py

```diff
@@ -1039,32 +1039,48 @@ def wrap(self, compound="atoms", center="com", box=None):
             if not all(s == 0.0):
                 o.atoms.translate(s)
 
-    def groupby(self, topattr):
+    def groupby(self, *topattrs):
         """Group together items in this group according to values of *topattr*
 
         Parameters
         ----------
         topattr: str
-           Topology attribute to group components by.
+           One or more topology attribute to group components by.
 
         Returns
         -------
         dict
-            Unique values of the topology attribute as keys, Groups as values.
+            Unique values of the multiple combinations of topology attributes 
+            as keys, Groups as values.
 
         Example
         -------
-        To group atoms with the same mass together::
+        To group atoms with the same residue name and mass together::
+
+          >>> ag.groupby('resnames', 'masses')
+          {'ALA': {1.008: <AtomGroup with 95 atoms>,
+            12.011: <AtomGroup with 57 atoms>,
+            14.007: <AtomGroup with 19 atoms>,
+            15.999: <AtomGroup with 19 atoms>},
+           'ARG': {1.008: <AtomGroup with 169 atoms>,
+           ...}
+          
+          >>> ag.groupby('resnames', 'masses')['ALA'][15.999]
+           <AtomGroup with 19 atoms>
+        
+        .. versionadded:: 
```

Comment:
The version number disappeared here. Also, you should add a line indicating that the function changed in version 0.18.0:

```
.. versionchanged:: 0.18.0

    The function accepts multiple attributes.
```